mule-extensions-api/src/test/java/org/mule/runtime/extension/api/NameUtilsTestCase.java

```diff
@@ -91,6 +92,13 @@ public void hyphenization() {
     assertThat("parameter-dsl-name", is(hyphenize("ParameterDSLName")));
   }
 
+  @Test
+  public void defaultNamespaceWithCamelCaseWordsSeparatedBySpaces() {
+    assertThat(defaultNamespace("AbCd Efgh"), is("abcd-efgh"));
```

Comment:
use real values, including (and similar to) the ones that caused the bug you're fixing

Reply:
Done.